tests/baselines/reference/catchClauseWithTypeAnnotation.errors.txt

```diff
@@ -2,13 +2,18 @@ tests/cases/conformance/statements/tryStatements/catchClauseWithTypeAnnotation.t
 tests/cases/conformance/statements/tryStatements/catchClauseWithTypeAnnotation.ts(18,37): error TS2339: Property 'foo' does not exist on type 'unknown'.
 tests/cases/conformance/statements/tryStatements/catchClauseWithTypeAnnotation.ts(19,23): error TS1196: Catch clause variable type annotation must be 'any' or 'unknown' if specified.
 tests/cases/conformance/statements/tryStatements/catchClauseWithTypeAnnotation.ts(20,23): error TS1196: Catch clause variable type annotation must be 'any' or 'unknown' if specified.
+tests/cases/conformance/statements/tryStatements/catchClauseWithTypeAnnotation.ts(25,23): error TS2339: Property 'toLowerCase' does not exist on type 'number'.

```

Comment:
Interesting; this seems to imply that this PR now enables annotations of non any/unknown type (though, we have the explicit error for it). Probably out of scope for this particular fix though, but I do think that's interesting.

Reply:
I reverted to `any` | `unknown`. Yes, this is out of scope, I was just experimenting with this behavior.